gtk4/src/auto/editable_label.rs

```diff
@@ -69,6 +69,10 @@ impl EditableLabel {
         }
     }
 
+    pub fn set_editing(&self, editing: bool) {
```

Comment:
needs a since annotation

Reply:
wth, where does this even come from https://gitlab.gnome.org/GNOME/gtk/-/blob/main/gtk/gtkeditablelabel.h

Reply:
https://gitlab.gnome.org/GNOME/gtk/-/blob/main/gtk/gtkeditablelabel.c#L376

Reply:
The property existed before, but looks like it was changed to readwrite instead of readable, so I think we need to guard that on the toml on our side

Reply:
Ended up copying the implementation to manually and explicitly only generating get/notify. We can't easily add a version guard only for the `set_` wrapper of the property.